oauthproxy.go

```diff
@@ -800,7 +800,7 @@ func (p *OAuthProxy) doOAuthStart(rw http.ResponseWriter, req *http.Request, ove
 	)
 	if p.provider.Data().CodeChallengeMethod != "" {
 		codeChallengeMethod = p.provider.Data().CodeChallengeMethod
-		codeVerifier, err = encryption.GenerateRandomASCIIString(96)
+		codeVerifier, err = encryption.GenerateCodeVerifierString(32)
```

Comment:
@vishvananda really sorry for the long wait :/


Why did you decrease the amount of random bytes?

Reply:
I see that a minimum length of 43 characters is mentioned in the RFC. Which equals 32 bytes. But I would prefer that we rather take the maximum for security and go with 128 characters which should equal 96 bytes like before.

Reply:
That's fine. I'll default it to 128 instead